build/blocks/block-coauthor-avatar/block.json

```diff
@@ -0,0 +1,60 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/block.json",
+  "apiVersion": 2,
+  "name": "cap/coauthor-avatar",
```

Comment:
Yes.

I would then like to shorten the second half to avoid adding 12 characters to the CSS class for each block.

This one would be `co-authors-plus/avatar` with the CSS class `wp-block-co-authors-plus-avatar`

I used the prefix `cap/coauthor-` elsewhere in the code to identify which blocks need the author context provided to them in the author archive, but that can be done another way.

Reply:
> I would then like to shorten the second half

Works for me.

Reply:
This was addressed in https://github.com/Automattic/Co-Authors-Plus/pull/997/commits/e7048219d232f3baf9e48aee37539433d0904fcf